apiserver/pkg/auth/auth.go

```diff
@@ -148,17 +153,17 @@ func AuthInterceptor(needAuth bool, oidcVerifier *oidc.IDTokenVerifier, verb, re
 				})
 				return
 			}
+			ctx.Request = ctx.Request.WithContext(context.WithValue(ctx.Request.Context(), UserNameContextKey, userName))
 		}
 
 		// for graphql query
-		ctx1 := context.WithValue(ctx.Request.Context(), idtokenKey{}, rawToken)
-		ctx.Request = ctx.Request.WithContext(ctx1)
+		ctx.Request = ctx.Request.WithContext(context.WithValue(ctx.Request.Context(), UserNameContextKey, rawToken))
```

Comment:
Context will always be `rawToken` if you do not return.

Reply:
oh, there is a typo, I should use idTokenContextKey when store rawToken